README.md

```diff
@@ -55,7 +55,26 @@ Then configure the rules you want to use under the rules section.
 - no-untranslated-polyglots: see [documentation](./docs/rules/no-untranslated-polyglots.md).
 - no-untrimmed-polyglots: see [documentation](./docs/rules/no-untrimmed-polyglots.md).
 
+## Contributing
 
+Run all tests with:
 
+```bash
+npm test
+```
+
+Run only a specific test with:
+
+```bash
+mocha tests/lib/rules/only-literal-polyglots.js  --reporter progress
+```
 
+Add a new rule with [Yeoman's generator-eslint](https://www.npmjs.com/package/generator-eslint).
 
+Release a new version by:
+1. running the tests
+2. changing the `version` property in `package.json`
+3. running `npm publish`
+4. committing the new `package.json`
+5. running `git tag --annotate <x.y.z> master`
```

Comment:
1. in markdown, only use `1.` to avoid having to change all the following lines in case of removal
1. no need to `git tag` as a GH release adds a tag if needed